components/board/board.go

```diff
@@ -93,17 +84,6 @@ type Board interface {
 	WriteAnalog(ctx context.Context, pin string, value int32, extra map[string]interface{}) error
 }
 
-// A LocalBoard represents a Board where you can request SPIs and I2Cs by name.
-type LocalBoard interface {
```

Comment:
This is the most breaking change for the modular boards, so we should update their constructors to return a `board.Board` object asap when this is released.